aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -203,5 +204,48 @@ Tensor& logical_xor_(Tensor& self, const Tensor& other) {
   return native::logical_xor_out(self, self, other);
 }
 
+Tensor& lt_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(result.scalar_type() == kBool,
+      "The output tensor of lt must be a bool.");
+  auto iter = TensorIterator::comparison_op(result, self, other, true);
```

Comment:
Nit: add a comment for boolean args: `TensorIterator::comparison_op(result, self, other, /*check_mem_overlap=*/true);`

Reply:
Added